packages/config/config/devices/0x010f/fgs223.json

```
@@ -232,478 +208,232 @@
-			"label": "Reaction to Flood Alarm",
-			"description": "This parameter determines how the device will react to Flood Alarm frame.",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 3,
-			"defaultValue": 2,
-			"unsigned": true,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "Ignored",
-					"value": 0
-				},
-				{
-					"label": "Turn On",
-					"value": 1
-				},
-				{
-					"label": "Turn OFF",
-					"value": 2
-				},
-				{
-					"label": "Flash",
-					"value": 3
-				}
-			]
+			"$import": "templates/fibaro_template.json#water_flood_alarm_response"
 		},
 		"42": {
-			"label": "Reaction to CO/CO2/Smoke Alarm",
-			"description": "This parameter determines how the device will react to CO/CO2/Smoke Alarm frame.",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 3,
-			"defaultValue": 3,
-			"unsigned": true,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "Ignored",
-					"value": 0
-				},
-				{
-					"label": "Turn On",
-					"value": 1
-				},
-				{
-					"label": "Turn OFF",
-					"value": 2
-				},
-				{
-					"label": "Flash",
-					"value": 3
-				}
-			]
+			"$import": "templates/fibaro_template.json#smoke_alarm_response"
 		},
 		"43": {
-			"label": "Reaction to Heat Alarm",
-			"description": "This parameter determines how the device will react to Heat alarm frame.",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 3,
-			"defaultValue": 1,
-			"unsigned": true,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "Ignored",
-					"value": 0
-				},
-				{
-					"label": "Turn On",
-					"value": 1
-				},
-				{
-					"label": "Turn OFF",
-					"value": 2
-				},
-				{
-					"label": "Flash",
-					"value": 3
-				}
-			]
+			"$import": "templates/fibaro_template.json#temperature_alarm_response"
 		},
 		"44": {
-			"label": "Flashing alarm duration",
-			"description": "This parameter allows to set duration of flashing alarm mode.",
-			"valueSize": 2,
-			"unit": "seconds",
-			"minValue": 1,
-			"maxValue": 32000,
-			"defaultValue": 600,
-			"unsigned": true
+			"$import": "templates/fibaro_template.json#alarm_state_time",
+			"maxValue": 32000
 		},
 		"50": {
-			"label": "First channel - power reports",
-			"description": "This parameter determines the minimum change in consumed power that will result in sending new power report to the main controller.",
-			"valueSize": 2,
-			"unit": "%",
-			"minValue": 1,
-			"maxValue": 100,
+			"$import": "templates/fibaro_template.json#active_power_report_threshold",
 			"defaultValue": 20,
-			"unsigned": true
+			"label": "First Channel - Active Power Report: Threshold"
 		},
 		"51": {
-			"label": "First channel - minimal time between power reports",
-			"description": "This parameter determines minimum time that has to elapse before sending new power report to the main controller.",
-			"valueSize": 2,
+			"label": "First Channel - Active Power Report: Minimum Time Between Reports",
```

Wouldn't this just be
```suggestion
			"label": "First Channel - Active Power Report: Interval",
```
in simpler words?

---

I don't think minimum time is the same as interval. At least to me naming it interval would make the difference between this and parameter 58 `Periodic Active Power Reports` less clear

---

Okay